requests-for-comments/marketing-and-content-team.md

```diff
@@ -0,0 +1,50 @@
+# RFC: Marketing & Content Team Questions
+
+As per [my PR on the handbook](https://github.com/PostHog/posthog.com/pull/6010) the Marketing team is potentially evolving to take greater ownership of all content, specifically working with Website & Docs to ensure the docs are up-to-date and maintained to a high-standard.
+
+This poses a bunch of questions about the team, who does what, and how we balance this work against existing marketing work. Here are some questions and answers in no particular order (please add your own):
+
```

Comment:
The first question here, to me, is to clarify exactly what is meant by 'ensuring that the docs are up to date'. 

Are we meaning by this that, whenever a new feature is added or introduced, it is on the marketing team to create that content for the docs from scratch, and to keep our docs constantly updated? Or is it merely that we're encouraging ourselves to improve the docs as and when we are able, to work with W+Ds and float/try ideas for improving the docs?

**If it's the former,** I think we need to consider ways to greatly strengthen the ways marketing works with engineering teams so we can achieve this. We'd also need to recognise that only Lior and Ian are technically strong enough to create much of the content and they may lose a lot of bandwidth. Existentially, there's also a risk with having too much of a 'marketing touch' in the docs (we could offset it, but it's there). It's potentially worth having a clear conversation about the value of hiring a dedicated technical docs writer to replace Paul, rather than spreading that (and his other work) across other people. 

**If it's the latter,** I'd argue we're already doing this - Lior has already been active in docs work, I've added new docs sections and routinely approved a lot of the content from some teams, Ian just updated the HogQL docs, etc. In that situation this RFC should present a proposal for how we further that direction.

Reply:
Additional: Agreed, it's worth exploring if we should hire another writer. I almost added that as a question but, given hiring takes a while and we've only recently added Lior, my gut is we should wait until we're confident about what kind of person we need / if we need someone.

Reply:
> Are we meaning by this that, whenever a new feature is added or introduced, it is on the marketing team to create that content for the docs from scratch, and to keep our docs constantly updated?

I see this more as the marketing reps are the responsible party to a) chase an engineer to make sure that _new_ features are documented when we announce them (especially if the technical details are beyond marketing's technical understanding), and b) when we become aware of holes/out-of-date content in the docs, that marketing either updates them or again chases an engineer to get the answers needed or have them update the docs directly.